source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRIBaseCardElementRenderer.h

```diff
@@ -17,7 +17,7 @@
 
 - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
            rootView:(ACRView *)rootView
-             inputs:(NSArray *)inputs
```

Comment:
Is this a breaking change do you know? What is the impact of updating this interface from NSArray to NSMutableArray?

Reply:
In Swift, this will change the signature of the method. Any existing custom renderers that had this method overridden will get an error saying "there is no method to override" since the signature has changed.

Reply:
Thanks for letting us know.

---
In reply to: [407759749](https://github.com/microsoft/AdaptiveCards/pull/3912#discussion_r407759749) [](ancestors = 407759749)

Reply:
for objective-c users such as Teams, this change will not have an impact for them. For swift, it can be a breaking changes as Johnny mentioned. 

---
In reply to: [407452344](https://github.com/microsoft/AdaptiveCards/pull/3912#discussion_r407452344) [](ancestors = 407452344)

Reply:
Thanks! This should likely be added with these notes in our release notes for 1.2.9 (@Paulcam206 this is another example of technically a breaking change for some consumers, but not that high impact so we're sticking with a patch update here)